Tests/BaseTestCase.swift

```diff
@@ -126,4 +126,17 @@ class BaseTestCase: XCTestCase {
 
         waitForExpectations(timeout: timeout)
     }
+    
+    /// Check memory leaks for an object when a test operation has been finished.
+    ///
+    /// - Parameters:
+    ///   - object: The testing object for each class.
+    ///   - file: Which file that failure test happens.
+    ///   - line: Which line that failure test happens.
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
```

Comment:
Can you add explicit watchOS and visionOS support as well? I think the other OSes are on their own.

Reply:
I have added watchOS but for visionOS I think there is no explicit need to be added.

```
@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
@preconcurrency public func addTeardownBlock(_ block: @escaping @MainActor @Sendable () throws -> Void)
```